core/driver/sources/src/main/scala/org/virtuslab/ideprobe/config/DriverConfig.scala

```diff
@@ -1,12 +1,13 @@
 package org.virtuslab.ideprobe.config
 
-import org.virtuslab.ideprobe.config.DriverConfig.LaunchParameters
+import org.virtuslab.ideprobe.config.DriverConfig.{LaunchParameters, ScreenConfig}
 
 import scala.concurrent.duration._
 
 case class DriverConfig(
     launch: LaunchParameters = LaunchParameters(),
     check: CheckConfig = CheckConfig(),
+    screen: ScreenConfig = ScreenConfig(),
     headless: Boolean = false,
```

Comment:
Actually... what's the use of this option @lukaszwawrzyk, given that we've got `IDEPROBE_DISPLAY` (which, by the way, is not reflected in any way in HOCON)?

Reply:
This only passes `-Djava.awt.headless=true` to the VM IIRC... but how does it compare in its use to Xvfb?

Reply:
It is orthogonal thing. IntelliJ can run in headless mode or UI mode, you can use it on local machine or on CI. If it is enabled, you don't need xvfb on CI.

Reply:
Oh, so with headless mode... no UI gets allocated at all, right? so what are the actions you can even do/test then?

Reply:
Well, you can invoke endpoints in ide-probe, I think remote robot might also work. You should generally be able to run many things, intelliJ code has some `if`s here and there to behave differently in headless mode. I think if opening project doesn't open any strange modals it will work, I think we used it to run pants import and extract shared indexes using a special intellij app.